frontend/src/components/common/form/Typeahead.tsx

```diff
@@ -58,11 +61,15 @@ export function TypeaheadField<T extends TypeaheadModel>({
         {...rest}
         inputProps={{ ...rest.inputProps, name: name, id: `${name}-field` }}
         isInvalid={hasError as any}
-        isValid={!filter && isValid}
+        isValid={!hideValidation && isValid}
         selected={getOptionByValue(getIn(values, name))}
-        onChange={(newValues: T[]) => {
-          setFieldValue(name, getIn(newValues[0], 'value') ?? newValues[0]);
-        }}
+        onChange={
```

Comment:
The default uses the value, but it can now be overridden?

Reply:
Correct, has its' initial implementation still so location still behaves as expected but for our agency case we override it in order to function as expected